handlers/line/line.go

```diff
@@ -32,6 +32,14 @@ var (
 	signatureHeader = "X-Line-Signature"
 )
 
+// see https://core.telegram.org/bots/api#sending-files
+var mediaSupport = map[handlers.MediaType]handlers.MediaTypeSupport{
+	handlers.MediaTypeImage:       {Types: []string{"image/jpe", "image/jpg", "image/jpeg", "image/png"}, MaxBytes: 10 * 1024 * 1024},
+	handlers.MediaTypeAudio:       {Types: []string{"audio/mp4", "audio/m4a"}, MaxBytes: 200 * 1024 * 1024},
```

Comment:
on the rapidpro side we always correct `audio/m4a` to `audio/mp4` so don't think this is needed

Reply:
updated